src/javascript/RichTextCKEditor5/RichTextCKEditor5-overrides.css

```diff
@@ -13,3 +13,7 @@
     padding-bottom: 0 !important;
     width: var(--ck-link-panel-width) !important;
 }
+
+.ck-clipboard-drop-target-line {
+    z-index: var(--ck-z-modal);
```

Comment:
Need to be in the same z-index as the modal to show the drop line when dragging images